packages/path-inputs/src/button.tsx

```diff
@@ -226,6 +237,7 @@ export function Button(props: ButtonProps) {
         [classes.btnActive]: props.isActive && !isCTA,
         [classes.btnCTAActive]: props.isActive && isCTA,
         [classes.btnIconOnly]: props.icon && !props.label && !props.isProcessing,
+        [className || '']: true,
```

Comment:
you can provide a simple `className` as a string. `classnames` library takes care for undefined values
```suggestion
        className,
```